protocols/gossipsub/src/config.rs

```diff
@@ -775,20 +775,6 @@ impl ConfigBuilder {
         self
     }
 
-    /// The maximum byte size for each gossip for a given topic. (default is
-    /// [`Self::max_transmit_size`]).
-    pub fn max_transmit_size_for_topic(
-        &mut self,
-        max_transmit_size: usize,
-        topic: TopicHash,
-    ) -> &mut Self {
-        self.config
-            .protocol
-            .max_transmit_sizes
-            .insert(topic, max_transmit_size);
-        self
-    }
-
```

Comment:
I'd prefer if we keep this one. Because this is the `ConfigBuilder`, none of the other functions are prefixed with `set_` (apart from one).

Reply:
It was not about `set` for me, having `topic` in the beginning really helps: they only have limited settings and that way it's easier to see `topic_` settings.